composer.json

```diff
@@ -8,24 +8,26 @@
     "require": {
         "drupal/core": "^8.6",
         "drupal/field_group": "^1.0",
+        "drupal/maxlength": "~1.0@beta",
         "easyrdf/easyrdf": "0.10.0-alpha.1 as 0.9.1",
         "openeuropa/rdf_skos": "dev-master",
-        "openeuropa/oe_media": "~0.2.0",
+        "openeuropa/oe_media": "~0.3",
```

Comment:
Why do we require "openeuropa/oe_media" here? Isn't the ticket about limiting text field length?

Reply:
It was already there to support media fields.